posts/20240125/index.qmd

```diff
@@ -0,0 +1,191 @@
+---
+title: "Direct Preference Optimization: Your Language Model is Secretly a Reward Model"
+author: "Kshitij Aggarwal"
+date: "2024-01-25"
+image: fig3.png
+description: This paper proposes a new technique to align LLMs with human preferences without using RL. This method is more robust and shows better performance over models trained with RLHF.   
+categories: [paper]
+---
+
+:::{.callout-note}
+- Paper: [Direct Preference Optimization: Your Language Model is Secretly a Reward Model](https://arxiv.org/abs/2305.18290)
+- Presenter: [Kshitij](https://www.linkedin.com/in/kshitij13/)
+- Attendees: [Kate](https://www.linkedin.com/in/kate-behrman/), [Hongsup](https://www.linkedin.com/in/hongsupshin/), [Todd](https://www.linkedin.com/in/troazen/), [Meghann](https://www.linkedin.com/in/meghann-agarwal/)
+:::
+
+## Why this paper?
+
+Following the latest advancements in LLM training, I've been interested in Direct Preference Optimization (DPO), a technique emerging as a superior method. The buzz around DPO was further amplified by Andrew Ng's [tweet](https://twitter.com/AndrewYNg/status/1745516258697863259?lang=en), praising the method for its simplicity and profound insight. 
+
+ <!-- DPO appeared to be the next best technique. It was being mentioned at various forums, used in new models (Mixtral?). Therefore, I was curious to see how it was different and better than PPO, that was supposed to be its predesessor. Then finally, I saw the [tweet](https://twitter.com/AndrewYNg/status/1745516258697863259?lang=en) by Andrew Ng, applauding the paper for it's simplicity and profound insight, and so I wanted to dive deeper into it.  -->
```

Comment:
I think text inside <> is not rendered. I was wondering whether you just left it as a note in the raw text or whether you want this to be shown differently from a regular text paragraph. If it's the latter, check this out: https://quarto.org/docs/authoring/markdown-basics.html#callout-blocks (and other markdown features)

Reply:
Those are just commented out lines. I don't want those to get rendered.